data/json/recipes/weapon/mods.json

```diff
@@ -23,6 +23,19 @@
     ]
   },
   {
+    "result": "crafted_suppressor",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "mechanics",
+    "difficulty": 1,
+    "skills_required": [ "gun", 1 ],
+    "time": 650,
+    "autolearn": true,
+    "qualities": [ { "id": "SAW_M_FINE", "level": 1 } ],
+    "components": [ [ [ "filter_liquid", 1 ], [ "filter_liquid_makeshift", 1 ] ] ]
```

Comment:
> Suppressors are exceptionally useful

You mean in the lore?
Because in the game, except for some silent ammo types, they're more useful for the recoil drop than the loudness modifier.
Shotgun ones are the best example of that - shotgun suppressor is a makeshift ported barrel with some minor ear protection capabilities.

> If we make it too easy one of the major balancing points of guns (noise) is lost.

There isn't much to be lost there, as currently we have guns that are silent (pneumatic) and guns that can be heard from the other end of the bubble.

Default loudness (if not specified otherwise) is `max( max(damage, range, pierce)*3, recoil / 3)`. For 9mm, this is 78 - enough for the whole reality bubble and then some. For 00 shot, it's 160 - 40 points above upper cap for hordes (to prevent them from coming from the other side of the world).